src/test/java/org/apache/commons/codec/language/RefinedSoundexTest.java

```diff
@@ -93,4 +94,13 @@ public void testNewInstance2() {
     public void testNewInstance3() {
         assertEquals("D6043", new RefinedSoundex(RefinedSoundex.US_ENGLISH_MAPPING_STRING).soundex("dogs"));
     }
+
+    @Test
+    public void testInvalidSoundexCharacter() {
+        char[] invalid = new char[256];
+        for (int i = 0; i < 256; i++) {
```

Comment:
Hello @arthurscchan 
You'll want to remove use of the magic number here and say `invalid.length`.
TY!

Reply:
Have changed. Thanks.